### PR TITLE
Add release instructions for the uninstall tool

### PR DESCRIPTION
**Pull request by baronfel**
Documents the release process for the dotnet-core-uninstall tool, which we are all fleshing out for the first time today :)

### CODE REVIEW

File: src/dotnet-core-uninstall/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# dotnet-core-uninstall
+
+This project is a standalone uninstaller for Windows and MacOS.  It exists partially to ensure that a) it's easy to uninstall the SDK and b) SDKs that are required by various VS versions aren't also uninstalled.
```
Comment (joeloff) [inline src/dotnet-core-uninstall/README.md:3]:
```suggestion
This project is a standalone uninstaller for Windows and MacOS. It exists partially to ensure that a) it's easy to uninstall the SDK and b) SDKs that are required by various VS versions aren't also uninstalled.
```

**Review by joeloff — APPROVED** [review]

File: src/dotnet-core-uninstall/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# dotnet-core-uninstall
+
+This project is a standalone uninstaller for Windows and MacOS. It exists partially to ensure that a) it's easy to uninstall the SDK and b) SDKs that are required by various VS versions aren't also uninstalled.
```
Comment (nagilson) [inline src/dotnet-core-uninstall/README.md:3]:
```suggestion
This project is a standalone uninstaller for the .NET SDK on Windows and MacOS. It exists partially to ensure that a) it's easy to uninstall the SDK and b) SDKs that are required by various VS versions aren't also uninstalled.
```

File: src/dotnet-core-uninstall/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+2. Double click the msi file to install.
+3. Open a new command prompt.
+4. Check out the docs at https://aka.ms/dotnet-core-uninstall-docs
+5. Run the tool by using dotnet-core-uninstall.
+6. If you accidentally mess up your Visual Studio installation, use "Repair" in the Visual Studio installer.
+
+## On macOS:
+
+Download the tar.gz file below.
+Open terminal and change working directory to the directory with dotnet-core-uninstall.tar.gz.
```
Comment (nagilson) [inline src/dotnet-core-uninstall/README.md:41]:
```suggestion
Open a terminal and change the working directory to the directory with dotnet-core-uninstall.tar.gz.
```

File: src/dotnet-core-uninstall/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+# Releasing the tool
+
+To release the tool, you need to do the following:
+
+* bump the version of the tool in eng/Versions.props
+* send a PR with this change and get it merged
+* wait for the associated build pipeline in VSTS to complete
+* get the version generated from the build pipeline and use it to create a tag in git
+* use that tag to create a github release. Add release notes with
```
Comment (nagilson) [inline src/dotnet-core-uninstall/README.md:13]:
```suggestion
* use that tag to create a GitHub release. Add release notes with
```

**Review by nagilson — COMMENTED** [review]
Thanks for making this!

**Pull request merged**